helm_deploy/values-train.yaml

```diff
@@ -6,7 +6,14 @@ generic-service:
   serviceAccountName: null
   ingress:
     host: locations-inside-prison-api-train.hmpps.service.justice.gov.uk
-
+    annotations:
+      # Secure the reset-training endpoint from outside of the Kubernetes ingress
+      nginx.ingress.kubernetes.io/server-snippet: |
+        server_tokens off;
+        location /reset-training {
```

Comment:
Prevents access from ingress

Reply:
probably best to prevent access in all envs rather than just in the training one